scripts/post_release_prep.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -x
+
+# To be executed after release has been deployed to prod
+# pre-requisites: install github CLI
+# - github documentation: https://github.com/cli/cli#installation
+
+git fetch --all --tags
+git checkout release-al2
+git reset --hard origin/release-al2
+
+# get latest release version
+release_version=$(git ls-remote --tags --sort=creatordate | grep -o 'v.*' | tail -1)
+echo ${release_version}
+
```

Comment:
Im thinking if its possible to add a pause and question before continuing with release?

```
echo "Publishing $release_version, would you like to proceed? [y,n]"

read publishInput

if [[ $publishInput != 'y' ]]
then
    echo "Aborting"
    exit 0
fi

echo "done something"
```